core/domain/acl_decorators_test.py

```diff
@@ -219,9 +219,10 @@ def setUp(self):
         rights_manager.publish_collection(self.owner, self.published_col_id)
 
     def test_guest_is_redirected_to_login_page(self):
```

Comment:
Some of these should be renamed, right, since a redirect isn't happening?

Also, we should have tests for both the HTML response case and the JSON response case.

Reply:
Done.